src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -183,11 +188,11 @@ export function transform(formConfig, form) {
   );
 
   const claimedConditions = clonedData.ratedDisabilities
-    ? clonedData.ratedDisabilities.map(d => d.name)
+    ? clonedData.ratedDisabilities.map(d => d.name.toLowerCase())
     : [];
   if (clonedData.newDisabilities) {
     clonedData.newDisabilities.forEach(d =>
-      claimedConditions.push(d.condition),
+      claimedConditions.push(d.condition.toLowerCase()),
```

Comment:
I keep making everything lower case because I was running into a bug where the new condition name as I entered it was `PTSD (post traumatic stress disorder)`, but the selection in `powDisabilities` was `PTSD (post Traumatic Stress Disorder)`. Not sure what the story is there exactly, but rather than changing the name, I just compared all lower cases.

We may want to change it up in `addCheckboxPerNewDisability` (or whatever). I'm not sure.